streams-tests/shared/src/test/scala/zio/stream/experimental/ZSinkSpec.scala

```diff
@@ -20,6 +20,14 @@ object ZSinkSpec extends ZIOBaseSpec {
   def spec: ZSpec[Environment, Failure] = {
     suite("ZSinkSpec")(
       suite("Constructors")(
+        suite("collectAllN")(
```

Comment:
Same re tests from ZTransducerSpec :-)

Reply:
Done in 239c55418cd31a51a0416d96f8037cda6119f4b7. As discussed over chat, we have to leave with trailing chunks, therefore the nonapplicable tests are dropped.